envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/HostHeaderRewritingTest.kt

```diff
@@ -73,3 +74,10 @@ internal class HostHeaderRewritingTest : EnvoyControlTestConfiguration() {
         }
     }
 }
+
+fun ObjectAssert<Response>.hasHostHeaderWithValue(overriddenHostHeader: String): ObjectAssert<Response> {
```

Comment:
`hasHostHeaderWithValue` in `EnvoyControlTestConfiguration` is no longer used anywhere. You can remove it.

Reply:
it's now done!